From 412497b078ec452e6d8a081862bc895c029c7a44 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:12:33 -0400
Subject: [alsa] compress: remove the sample rate check

Message-id: <140972823573-15-patchset-jkysela@redhat.com>
Patchwork-id: 90715
O-Subject: [RHEL 7.1 PATCH 014/442] ALSA: compress: remove the sample rate check
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

commit f0e9c080 - "ALSA: compress: change the way sample rates are sent to
kernel" changed the way sample rates are sent. So now we don't need to check for
PCM_RATE_xxx in kernel

Signed-off-by: Vinod Koul <vinod.koul@intel.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 2aac06f787940543fb37bfdb982eb99431bc6094)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/core/compress_offload.c b/sound/core/compress_offload.c
index 9d518ac..7a20897 100644
--- a/sound/core/compress_offload.c
+++ b/sound/core/compress_offload.c
@@ -501,9 +501,6 @@ static int snd_compress_check_input(struct snd_compr_params *params)
  if (params->codec.ch_in == 0 || params->codec.ch_out == 0)
   return -EINVAL;
 
- if (!(params->codec.sample_rate & SNDRV_PCM_RATE_8000_192000))
-  return -EINVAL;
-
  return 0;
 }
 
-- 
1.7.1